FEAT: aarch64 - FAILS: emulated generates wrong zig1.wasm

MementoRC opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

regro-cf-autotick-bot committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.10.13.43.2

MementoRC committed:
feat: update to version 0.15.2 in rattler-build format

MementoRC committed:
feat: enable native aarch64 builds on native runners

   Zig on native aarch64 runners

   Changes:
   - Removed linux_aarch64 from build_platform cross-compilation mapping
   - Created native-cmake-linux-aarch64.sh build script (based on linux-64)
   - Updated build.sh to route aarch64 builds to native script
   - Removed cross-compilation specific patches for aarch64
   - Simplified aarch64 requirements (removed cross-compilation dependencies)
   - Removed __glibc ignore_run_exports for native aarch64

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

MementoRC committed:
fix: workaround zig1.wasm aarch64 assembly bug with ZIG_HOST_TARGET_TRIPLE

   zig1.wasm (WebAssembly stage1 compiler) has a bug generating
   aarch64 inline assembly, resulting in errors like:
     error: invalid register name for 't2'

   The solution: Force zig1.wasm to generate x86_64 code for the bootstrap phase
   (zig2.c and compiler_rt.c) by setting -DZIG_HOST_TARGET_TRIPLE=x86_64-linux-gnu.
   The final stage3 compiler will still properly target aarch64.

MementoRC committed:
(fix) azure are qemu runners. try github action

MementoRC commented:
obsoleted by fast cross-zig build
